apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreDbTest.cs

```diff
@@ -68,6 +68,24 @@ public void Lookup_NamespaceOnly()
             Assert.Equal("bar", (string)entity["foo"]);
         }
 
+        [Fact]
```

Comment:
It would be good to keep a test for each operation that ends up using the database ID, to check that existing code using the default still behaves appropriately. (Possibly just a single test to create a client, store an entity, query that entity, then delete it again.)

Reply:
Sure